src/kvstore/plugins/hbase/HBaseStore.cpp

```diff
@@ -199,6 +199,27 @@ ResultCode HBaseStore::prefix(GraphSpaceID spaceId,
 }
 
 
+ResultCode HBaseStore::rangeWithPrefix(GraphSpaceID spaceId,
+                                       PartitionID  partId,
+                                       const std::string& start,
+                                       const std::string& prefix,
+                                       std::unique_ptr<KVIterator>* storageIter) {
+    UNUSED(partId);
+    auto tableName = this->spaceIdToTableName(spaceId);
+    std::string startRowKey, endRowKey;
+    startRowKey = this->getRowKey(start);
+    endRowKey.reserve(kMaxRowKeyLength);
+    for (size_t n = 0; n < kMaxRowKeyLength - prefix.size(); n++) {
```

Comment:
size_t is unsigned, so take care about that if prefix.size() is bigger than kMaxRowKeyLength, is the compiler will convert the result (kMaxRowKeyLength - prefix.size()) to unsigned?

Reply:
Both `kMaxRowKeyLength` and `prefix.size()` is size_t, and the caller need make sure prefix is legal.

Reply:
even so, I think double check is needed here.